scripts/meta_analysis.py

```diff
@@ -580,12 +577,12 @@ def get_next_variant( studies : List[Study]) -> List[VariantData]:
                 del dats[i][j]
                 s.put_back(dats[i])
                 break
-            if not v.is_equal(first):
+            if not v.is_equal(first, equalize=False, flip_indels=s.flip_indels):
```

Comment:
and then in all these use the appropriate public method

Reply:
Resolved in https://github.com/FINNGEN/META_ANALYSIS/pull/46/commits/c3b166e151ad090fb795de67567017a2e124a818.